Makefile

```diff
@@ -168,10 +168,8 @@ ifndef ANDROID_SECRETS_IV
 	$(eval ANDROID_KEYSTORE_PASSWORD := $(shell echo ${${VARNAME}}))
 	$(eval VARNAME=ANDROID_KEY_PASSWORD_${ORG_UPPER})
 	$(eval ANDROID_KEY_PASSWORD := $(shell echo ${${VARNAME}}))
-	$(eval VARNAME=ANDROID_KEY_ALIAS_${ORG_UPPER})
-	$(eval ANDROID_KEY_ALIAS := $(shell echo ${${VARNAME}}))
-	$(eval VARNAME=ANDROID_KEYSTORE_PATH_${ORG_UPPER})
-	$(eval ANDROID_KEYSTORE_PATH := $(shell echo ${${VARNAME}}))
+	$(eval ANDROID_KEY_ALIAS := "medicmobile")
+	$(eval ANDROID_KEYSTORE_PATH := "${org}.keystore")
```

Comment:
Can remove these doc lines down below:

```
	$(info export ANDROID_KEYSTORE_PATH_$(ORG_UPPER)=$(ANDROID_KEYSTORE_PATH))
	$(info export ANDROID_KEY_ALIAS_$(ORG_UPPER)=$(ANDROID_KEY_ALIAS))
```

Reply:
Also, should be able to remove these variables from the [test-keystore.bats](https://github.com/medic/cht-android/blob/5e64289fe74a125f0118cdd7daaaa04833ae1ccc/src/test/bash/test-keystore.bats#L91) file (make sure to remove from both tests) without those tests failing now!